### PR TITLE
GetStringDistance

### PR DESCRIPTION
**Pull request by dorschw**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/44463

## Description
Removed the `Condition` tag
Added test playbook

## Screenshots
Paste here any images that will help the reviewer

## Does it break backward compatibility?
   - [x] Yes (but)
       - Further details: the behaviour caused by the `Condition` tag is quite impossible to use, so not really breaking.
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Review by DeanArbel — APPROVED** [review]

**Comment by DeanArbel** [comment]
@yaron-libman Can you please assign someone to review the docs?

**Comment by yaron-libman** [comment]
@tschanfeld The only current change is the RN, but @DeanArbel wanted us to give this a more thorough review since it hasn't been looked at in some time (possibly ever).

File: Packs/CommonScripts/Scripts/GetStringsDistance/GetStringsDistance.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+type: python
+subtype: python2
+tags:
+- server
+- phishing
+comment: Get the string distance between inputString and compareString (could be a
+  comma separated list) based on Levenshtein Distance algorithm.
+system: true
+args:
+- name: compareString
```
Comment (DeanArbel) [inline Packs/CommonScripts/Scripts/GetStringsDistance/GetStringsDistance.yml:15]: @dorschw Please add `isArray: true`.

**Comment by tschanfeld** [comment]
made minor edits, done

**Pull request merged**